presentation/src/main/res/values/strings.xml

```diff
@@ -431,6 +430,9 @@
     </plurals>
     <string name="notification_message_failed_title">Message not sent</string>
     <string name="notification_message_failed_text">The message to %s failed to send</string>
+    <string name="open_source_credit_title">Open Source Credit</string>
+    <string name="up_open_source_link_title">Unplugged Source Code</string>
```

Comment:
These three lines make all the above translation additions redundant.

Reply:
When I added these strings it asked me to which locals to add them so I chose all of them, so it will present these in all languages and will not have issues with missing strings